### PR TITLE
Add docs for persist blacklist

### PR DESCRIPTION
**Pull request by calvinjia**
Add docs on how to use persist blacklist

### CODE REVIEW

File: docs/en/advanced/Performance-Tuning.md [hunk 1]
```diff
@@ -222,7 +222,8 @@ The worker will asynchronously continue to read the remainder of the block witho
 The number of asynchronous threads used to finish reading partial blocks is set by the
 `alluxio.worker.network.async.cache.manager.threads.max` property.
 When large amounts of data are expected to be asynchronously cached concurrently, it may be helpful
-to increase this value to handle a higher workload.
+to increase this value to handle a higher workload. This is most commonly effective in cases where
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: New sentences should be on a new line

File: docs/en/advanced/Performance-Tuning.md [hunk 2]
```diff
@@ -277,6 +278,8 @@ alluxio.user.file.persist.on.rename=true
 # Determines the number of copies in Alluxio when files are not yet persisted, increase this to
 # a larger number to ensure fault tolerance in case of Alluxio worker failures
 alluxio.user.file.replication.durable=1
+# blacklist files which have _temporary in their path
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: can we list some examples that would match this? At first glance I wonder 

(using glob syntax)
- Is it the file extension? `**/*._temporary`?
- Is it a path component? `**/_temporary/**`?
- Is it a directory? `**/_temporary/*`?
- Is it the filename? `**/_temporary`?
- Is it anywhere in the path? `**_temporary**`?

File: docs/en/advanced/Performance-Tuning.md [hunk 3]
```diff
@@ -292,3 +295,20 @@ With this configuration, the protocol translates to the following:
 Overall, a copy and delete operation in the object store is saved, and the slow portion of writing
 to the object store is moved off the critical path.
 
+In some cases, the compute framework's commit protocol involves multiple renames or temporary files.
+Alluxio provides a mechanism for preventing files from being persisted by blacklisting a set of
+strings which are associated with temporary files. Any file which has any of the configured strings
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: newline for sentence

File: docs/en/advanced/Performance-Tuning.md [hunk 4]
```diff
@@ -292,3 +295,20 @@ With this configuration, the protocol translates to the following:
...
+as part of its path will not be considered for persist.
+
+For example, if
+
+```
+alluxio.master.persistence.blacklist=.staging,_temporary
+```
+
+Files such as `/data/_temporary/part-00001`, `/data/temporary.staging` will not be considered for
+persist. This works because eventually these temporary files will be deleted or renamed to
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: newline on sentence

File: docs/en/advanced/Performance-Tuning.md [hunk 5]
```diff
@@ -292,3 +295,20 @@ With this configuration, the protocol translates to the following:
...
+non-permanent files. Because `alluxio.user.file.persist.on.rename=true` is set, the files will be
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: `\n`

File: docs/en/advanced/Performance-Tuning.md [hunk 6]
```diff
@@ -292,3 +295,20 @@ With this configuration, the protocol translates to the following:
...
+considered for persist again when renamed. Note that persist on rename works for directories as well
```
Comment (ZacBlanco) [inline docs/en/advanced/Performance-Tuning.md]: `\n`

File: docs/en/advanced/Performance-Tuning.md (hunk 2 above)
Comment (calvinjia) [reply docs/en/advanced/Performance-Tuning.md]: We detail some examples later on, I thought it would be too wordy to put into a comment

Comment (ZacBlanco) [reply docs/en/advanced/Performance-Tuning.md]: yeah I noticed later on. Maybe it's just me, but if I wrote this I probably would have 

```
blacklist files which contain the string "_temporary" anywhere in their path
```

Though that's mainly because when I write things I tend to over-specify sometimes - I'll leave any change up to you

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.
